package.json

```diff
@@ -55,6 +55,7 @@
     "multihashes": "~0.4.14",
     "multihashing-async": "~0.5.2",
```

Comment:
Can you update [multiformats/js-multihashing-async](https://github.com/multiformats/js-multihashing-async) to the latest version, as it will already support promises? It is also used in utils, but you can callbackify those utils for now (address utils in an upcoming PR)

Reply:
@vasco-santos happy to do that in a separate PR